Cartesian_kernel/include/CGAL/Cartesian/Aff_transformation_3.h

```diff
@@ -141,15 +141,27 @@ class Aff_transformationC3
   operator()(const Direction_3 &d) const
   { return transform(d); }
 
+  Aff_transformation_3
+  transposed_inverse() const
```

Comment:
Stupid me, here it is.

Reply:
Maybe it's better just to make transpose() **public** instead of **protected** or even make a friend of `SM_decorator`?